Orange/widgets/unsupervised/owdistancetransformation.py

```diff
@@ -69,5 +74,25 @@ def commit(self):
             distances = inv(distances)
         self.send("Distances", distances)
 
+    def send_report(self):
+        items = OrderedDict()
+        norm, normopt = self.normalization_method, self.normalization_options
+        inv, invopt = self.inversion_method, self.inversion_options
+        if norm == 0:
```

Comment:
Yeah. This could also be done. So should I change it? However the first one is more naive.